test/e2e/storage/vsphere/k8s_utils.go

```diff
@@ -0,0 +1,240 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+type KubeletOpt string
+
+const (
+	NodeStateTimeout            = 1 * time.Minute
+	KStart           KubeletOpt = "start"
+	KStop            KubeletOpt = "stop"
+	KRestart         KubeletOpt = "restart"
+)
+
+// podExec wraps RunKubectl to execute a bash cmd in target pod
+func podExec(pod *v1.Pod, bashExec string) (string, error) {
```

Comment:
why not use the existing `podExec` in e2e/storage/ or move that podExec to framework/util.go or another common location? It is a very small wrapper but is redundant.

Reply:
I will work on it.
I will get podExec in framework/util.go and will use it in vsphere tests and e2e/storage too.